test/type-hint.vader

```diff
@@ -0,0 +1,104 @@
+# vim:set et sw=4 ts=4 tw=79:
+
+Given python (def foo no arg return str):
+    def foo() -> str:
+        pass
+
+Execute:
+    Pydocstring
+
+Expect python:
+    def foo() -> str:
+        """foo
+
+        return (str):
+        """
+        pass
+
+Given python (def foo with 1 arg return str):
+    def foo(sample_var) -> str:
+        pass
+Execute:
+    Pydocstring
+
+Expect python:
+    def foo(sample_var) -> str:
+        """foo
+
+        :param sample_var:
+
+        return (str):
+        """
+        pass
+
+Given python (def foo 2 args return str):
+    def foo(arg1, arg2) -> str:
+        pass
+Execute:
+    Pydocstring
+
+Expect python:
+    def foo(arg1, arg2) -> str:
+        """foo
+
+        :param arg1:
+        :param arg2:
+
+        return (str):
+        """
+        pass
+
+Given python (def foo 1 arg with type and return int):
+    def foo(n: int) -> int:
+        pass
+
+Execute:
+    Pydocstring
+
+Expect python:
+    def foo(n: int) -> int:
+        """foo
+
+        :param n (int):
+
+        return (int):
+        """
+        pass
+
+Given python (def foo 2 args with type and return int):
+    def foo(n: int, m: str) -> int:
+        pass
+
+Execute:
+    Pydocstring
+
+Expect python:
+    def foo(n: int, m: str) -> int:
+        """foo
+
+        :param n (int):
```

Comment:
Currently pydocstring.vim's default style is Sphinx style.
Can we do like following?
```python
"""foo

:param n:
:type n: int
:param m:
:type m: str
:rtype: int
```